pkg/scheduler/schedule_one.go

```diff
@@ -125,7 +125,7 @@ func (sched *Scheduler) schedulingCycle(
 	podInfo *framework.QueuedPodInfo,
 	start time.Time,
 	podsToActivate *framework.PodsToActivate,
-) (ScheduleResult, *framework.QueuedPodInfo, error) {
+) (ScheduleResult, *framework.QueuedPodInfo, *framework.Status) {
 
 	pod := podInfo.Pod
 	scheduleResult, err := sched.SchedulePod(ctx, fwk, state, pod)
```

Comment:
Is `ScheduleResult.reason` being used anywhere else? should we get rid of it?

Reply:
It's already removed below.